python/dataAPIutilities.py

```diff
@@ -4,48 +4,16 @@
 """
 
 from string import Template
+from vector.apps.DataAPI import mock_helper
 
 # Tag for the init, which we want to ignore
 TAG_FOR_INIT = "<<INIT>>"
 
-# list of functions not to be shown in the functions list
-FUNCTIONS_TO_IGNORE = {"coded_tests_driver", TAG_FOR_INIT}
 
-
-def functionCanBeMocked(functionObject):
-    """
-    # PCT-FIX-NEEDED - issue #7 - is_mockable not dependable
```

Comment:
Are all of these special cases supported by dataAPI now?
Are issues 7 and 8 fixed?  They are not marked as tested or fixed in confluence.

Reply:
I will check; I tested `vmock_bugs` and `vmock_examples` with the new logic and it didn't try to mock anything it shouldn't have done.